lib/mix/tasks/ecto.gen.repo.ex

```diff
@@ -73,6 +73,15 @@ defmodule Mix.Tasks.Ecto.Gen.Repo do
     """
   end
 
+  defp mixfile_loc(app) do
+    case Elixir.Version.compare(System.version, "1.4.0") do
+      :gt -> "lib/#{app}/application.ex"
+      :eq -> "lib/#{app}/application.ex"
+      :lt -> "lib/#{app}.ex"
+    end
+  end
+
+
```

Comment:
Do you happen to know why `mixfile_loc/1` doesn't seem to work even though i am on Elixir 1.4.2? I still get `lib/my_app.ex` when using `mix ecto.gen.repo`. (I didn't file an issue yet because (1) such a pity thing and (2) maybe it is me who is messing stuff up.)
Thanks!